lib/editor/util/validation.js

```diff
@@ -246,6 +252,35 @@ export function validate (
     case 'COLOR':
     case 'POSITIVE_INT':
     case 'POSITIVE_NUM':
+      if (isRequiredButEmpty) {
```

Comment:
Wouldn't this new check apply to all of the cases above? I think you only meant for it to apply to POSITIVE_INT and POSITIVE_NUM, right?

Reply:
oops

Reply:
Fixed that